release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2716,6 +2716,73 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, IPv6 networking services that operated in {rh-openstack-first} environments could not share an IPv6 load balancer that was configured with multiple services because of an issue that mistakenly marks an IPv6 load balancer as `Internal` to the cluster. With this release, IPv6 load balancers are no longer marked as `Internal` so that an IPv6 load balancer with multiple services can be shared among IPv6 networking services. (link:https://issues.redhat.com/browse/OCPBUGS-32246[*OCPBUGS-32246*])
+
+* Previously, the control plane machine sets (CPMS) did not allow template names for vSphere in a CPMS definition. With this release, a CPMS Operator fix allows template names for vSphere in the CPMS definition so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-32357[*OCPBUGS-32357*])
+
+* Previously, the control plane machine sets (CPMS) Operator was not correctly handling older {product-title} version configurations that had a vSphere definition in the infrastructure custom resource. This would cause cluster upgrade operations to fail and the CPMS Operator to remain in a `crashloopback` state. With this release, the cluster upgrade operations do not fail because of this issue. (link:https://issues.redhat.com/browse/OCPBUGS-32414[*OCPBUGS-32414*])
+
+* Previously, the image registry's Azure path fix job incorrectly required the presence of `AZURE_CLIENT_ID` and `TENANT_CLIENT_ID` parameters to function. This caused a valid configuration to throw an error message. With this release, adds a check to the Identity and Access Management (IAM) service account key to validate if these parameters are needed, so that a cluster upgrade operation no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-32396[*OCPBUGS-32396*])
+
+* Previously, a build pod that failed because of a memory limitation would have its pod status changed to `Error` instead of `OOMKilled`. This caused these pods to not be reported correctly. The issue would only occur on cgroup v2 nodes. With this release, a pod with a status of `OOMKilled` is correctly detected and reported. (link:https://issues.redhat.com/browse/OCPBUGS-32498[*OCPBUGS-32498*])
+
+[id="ocp-4-15-11-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-11-topology-page-node-limit-increase"]
+===== Increased the number of supported nodes on the *Topology* view
+
+* Previously, the {product-title} web console *Topology* view could only display a maximum of 100 nodes. If you attempted to view more than 100 nodes, the web console would output a `Loading is taking longer than expected.` error message. With this release, the `MAX_NODES_LIMIT` parameter for the web console is set to `200`, so that the web console can display a maximum of 200 nodes. (link:https://issues.redhat.com/browse/OCPBUGS-32340[*OCPBUGS-32340*]
```

Comment:
```suggestion
* Previously, the {product-title} web console *Topology* view could only display a maximum of 100 nodes. If you attempted to view more than 100 nodes, the web console would output a `Loading is taking longer than expected.` error message. With this release, the `MAX_NODES_LIMIT` parameter for the web console is set to `200`, so that the web console can display a maximum of 200 nodes. (link:https://issues.redhat.com/browse/OCPBUGS-32340[*OCPBUGS-32340*])
```